include/tscore/TSStatus.h

```diff
@@ -0,0 +1,90 @@
+/**
+  @file TSStatus.h
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+#pragma once
+
+#include <tscore/ink_assert.h>
+
+// Global status information for trafficserver
+//
+class TsStatus
+{
+private:
+  struct Data {
```

Comment:
Why have `Data` instead of static members of `TsStatus`?

Reply:
I'm using the pattern where non-const static data doesn't have to be defined separately in a .cc file.  When using that pattern, it's generally less verbose to group all the non-const data into a structure.  Although Brian's aversion to abbreviation make the difference in verbosity less.